src/main/index.js

```diff
@@ -143,6 +143,7 @@ function createWindow() {
     minWidth: 800,
     minHeight: 600,
     autoHideMenuBar: true,
+    titleBarStyle: 'hidden',
```

Comment:
This is preventing a draggable window. Should be a simple fix but definitely getting rid of the title bar. Need to explore how it looks on window/linux though